spec/shell-session-spec.coffee

```diff
@@ -1,12 +1,14 @@
 describe "Shell session grammar", ->
   grammar = null
+  grammar_session = null
 
   beforeEach ->
     waitsForPromise ->
       atom.packages.activatePackage("language-shellscript")
 
     runs ->
-      grammar = atom.grammars.grammarForScopeName("text.shell-session")
+      grammar_session = atom.grammars.grammarForScopeName("text.shell-session")
```

Comment:
@mnquintana Yep, [you're right](https://atom.io/docs/v0.186.0/contributing), sorry about that.
Do you want me to open a PR or can you fix it directly?